src/components/Form/CrossPodQueryForm.jsx

```diff
@@ -76,42 +89,39 @@ const CrossPodQueryForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
-  /* eslint-disable jsx-a11y/label-has-associated-control */
   return (
     <FormSection
       title="Cross Pod Search"
       state={state}
       statusType="Search status"
       defaultMessage="To be searched..."
     >
-      <form onSubmit={handleCrossPodQuery} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="cross-search-doc">Search document from username: </label>
-          <br />
-          <br />
-          <input
-            id="cross-search-doc"
-            size="25"
-            name="crossPodQuery"
-            {...username}
-            placeholder={selectedUser}
-          />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="cross-search-doctype">Select document type to search: </label>
-          <DocumentSelection htmlId="cross-search-doctype" />{' '}
-          <button disabled={state.processing} type="submit">
-            Search Pod
-          </button>
-        </div>
-      </form>
+      <Box display="flex" justifyContent="center">
+        <form onSubmit={handleCrossPodQuery} autoComplete="off">
+          <FormControl>
+            <TextField
+              id="cross-search-doc"
+              name="crossPodQuery"
+              {...username}
+              placeholder={selectedUser}
+              label="Enter username"
+              required
+            />
+          </FormControl>
+          <InputLabel id="cross-search-doctype">
```

Comment:
Already tested, this `<InputLabel>`'s `id` needs to be `htmlFor` to get rid of blue warnings in dev tools. Currently, it is not linked to what it's trying to label.

Reply:
Getting rid of the incorrect labels id with the correct htmlFor attribute.